packages/block-library/src/index.native.js

```diff
@@ -261,10 +261,13 @@ export const registerCoreBlocks = () => {
  * than 0, a "new" badge is displayed on the block type within the block
  * inserter.
  *
+ * With the below example, the Audio block will be displayed as "new" until it
+ * reaches 40 impressions.
```

Comment:
The impression count set here is decremented on various actions, e.g. block inserter closed ([decremented by 1](https://github.com/WordPress/gutenberg/blob/30c536310124e37bf7118bdc53b1621c2850ca59/packages/block-editor/src/components/inserter/index.native.js#L196)), block type inserted ([set to 0](https://github.com/WordPress/gutenberg/blob/30c536310124e37bf7118bdc53b1621c2850ca59/packages/block-editor/src/components/inserter/hooks/use-block-type-impressions.native.js#L32)). When the impression count for a block is [not greater than zero](https://github.com/WordPress/gutenberg/blob/30c536310124e37bf7118bdc53b1621c2850ca59/packages/block-editor/src/components/inserter/hooks/use-block-type-impressions.native.js#L26), it is no longer considered "new." 
```suggestion
 * With the below example, the Audio block will be displayed as "new" until its
 * impression count reaches 0, which occurs by various actions decrementing
 * the impression count.
```

Reply:
Good point, I forgot that it was set to 0 when being inserted. Thanks for the call out 🙇 !